apps/frontend/src/layout/AppHeader.style.ts

```diff
@@ -0,0 +1,55 @@
+import styled from "styled-components";
+import { Layout, Input, Button } from "antd";
+const { Header } = Layout;
+
+// Styled Components
+export const PromoBanner = styled.div`
+    background-color: ${props => props.theme.color.div.backgroundColor};
```

Comment:
```suggestion
    background-color: ${props => props.theme.color.backgroundColor};
```

You should use bg color only

Reply:
![image](https://github.com/user-attachments/assets/90034abc-fa34-4919-93b1-fab3d1d511e8)

In the type StyledComponentTheme's color properties, since I thought there will be different need of color/background-color for differences kinds of component: div (canvas), button, input (the background of the input field), etc, I created additional props inside color.

If it is possible that we can define a general color scheme for all components, I will merge all props of color into one. But I need more elaborate on the structure of the color props that you intend to use. Or if, only the name div is confusing, I may change it to canvas instead

Reply:
In a color palete, there are only a few colors, about 10 - 15, that will use for the whole app. Don't fail into the over-engineering hole =)

Reply:
Damn it! Failed already :)

Reply:
Right, I will reduce into a couple of color theme (each for light/dark theme) such as: canvasBackground, titleTextColor, primaryText, secondaryText, primaryButtonBackground, primaryButtonText, secondaryButtonBackground, secondaryButtonText, inputFormBackground, etc.

The color palette will shrink to about 10-15 color code as you said